lib/OpenQA/WebAPI.pm

```diff
@@ -393,6 +396,10 @@ sub startup {
     $api_public_r->get('job_templates/:job_template_id')->name('apiv1_job_template')->to('job_template#list');
     $api_ra->delete('job_templates/:job_template_id')->to('job_template#destroy');
 
+    # api/v1/job_templates_scheduling
+    $api_public_r->get('job_templates_scheduling/:id')->name('apiv1_job_templates_schedules')
```

Comment:
It makes sense to add it to the API. But at this point I wouldn't consider it stable. So better put it under some `/experimental` prefix for now.